### PR TITLE
updated event version to v1 in debug pod monitor

### PR DESCRIPTION
**Pull request by facchettos**
### Which issue this PR addresses:
https://issues.redhat.com/browse/ARO-2650
events `v1beta1` are being removed in kubernetes 1.25. This updates the events we are using in the debugpod monitor.  

Fixes

### What this PR does / why we need it:
see above

### Test plan for issue:
existing tests should be passing still

### CODE REVIEW

**Review by tiguelu — APPROVED** [review]
LGTM

**Review by yehornaumenko — APPROVED** [review]

**Review by jaitaiwan — APPROVED** [review]
LGTM

File: pkg/monitor/cluster/debugpods.go [hunk 1]
```diff
@@ -23,16 +24,16 @@ import (
 // in order to detect recent events of debug-pod creation.
 func (mon *Monitor) emitDebugPodsCount(ctx context.Context) error {
 	m := map[string]string{
-		"reason":              "Started",
-		"involvedObject.kind": "Pod",
+		"reason":         "Started",
+		"regarding.kind": "Pod",
```
Comment (s-amann) [inline pkg/monitor/cluster/debugpods.go:28]: do we have any monitors keying off `involvedObject.kind`?

**Review by s-amann — APPROVED** [review]

**Review by petrkotas — APPROVED** [review]
LGTM, the change is updating the key in the k8s. No metric name is changed.

**Pull request merged**